src/main/java/hudson/plugins/tics/AbstractApiCall.java

```diff
@@ -51,6 +55,16 @@ protected final CloseableHttpClient createHttpClient() throws KeyManagementExcep
             credsProvider.setCredentials(AuthScope.ANY, new UsernamePasswordCredentials(username, password));
             builder = builder.setDefaultCredentialsProvider(credsProvider);
         }
+
+        Jenkins jenkins = Jenkins.get();
+        if (jenkins != null) {
+            ProxyConfiguration proxy = jenkins.proxy;
+            if (proxy != null) {
+                logger.println("Using proxy: " + proxy.name + ":" + proxy.port);
+                HttpHost hostProxy = new HttpHost(proxy.name, proxy.port);
+                builder = builder.setProxy(hostProxy);
```

Comment:
What about proxy username & password that you can configure in Jenkins (see here: https://wiki.jenkins.io/JENKINS/JenkinsBehindProxy.htm) ? These do not seem to be used.